lib/polyn/transporters/internal.rb

```diff
@@ -32,6 +32,9 @@ def acknowledge; end
 
       def initialize(transit, options = {})
         super(transit, options)
+        if Object.const_defined?("::Rails")
+          logger.warn("WARNING, the internal transporter is not supported within Rails.")
+        end
```

Comment:
If we are calling this `polyn-ruby` we shouldn't be checking for rails. Otherwirse we should call this gem `polyn-rails`